nbbuild/build.xml

```diff
@@ -1968,6 +1968,7 @@ It is possible to use -Ddebug.port=3234 -Ddebug.pause=y to start the system in d
                 <exclude name="diff/test/unit/src/org/netbeans/modules/diff/builtin/visualizer/data/**" /> <!--test data-->
                 <exclude name="utilities/test/unit/src/org/netbeans/modules/openfile/resources/recent_files/*" /> <!--test data-->
                 <exclude name="**/*.url" /> <!--no degree of creativity-->
+		<exclude name="spi.navigator/test/unit/src/org/netbeans/modules/navigator/resources/lastsel/file.lastsel_mime1" /> <!--empty file-->    
```

Comment:
OK.

Reply:
I've merged now but note how on this line you've added some whitespace at the end :-|

Reply:
Thanks for the comment. I'll consider being more careful of the whitespaces and tabs later on for sure :-)